apps/docs/app/features/root-providers/RootProviders.tsx

```diff
@@ -9,8 +10,11 @@ type RootProvidersProps = { children: React.ReactNode };
  */
 export const RootProviders = ({ children }: RootProvidersProps) => {
   return (
-    <SporProvider language={Language.NorwegianBokmal}>
-      <PortableTextProvider>{children}</PortableTextProvider>
-    </SporProvider>
+    <SSRProvider>
```

Comment:
Mens jeg forgjeves leste react-aria docs kom jeg over at vi manglet denne. Det er i utgangspunktet ikke noe problem for oss enda, siden vi bare viser date- og timepickers i kontekst av klient-side komponenten som viser previews av komponenter, men det er nå en gang best practice.